### PR TITLE
Update cisagov_N.yml

### PR DESCRIPTION
**Pull request by kyle-ni**
#  #

## 🗣 Description ##

- Updated status of NI Products
- fixed typo in 'notes'
- removed duplicate entry

## ✅ Pre-approval checklist ##

- [ ] This PR has an informative and human-readable title.
- [ ] Changes are limited to a single goal - _eschew scope creep!_
- [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
- [ ] All relevant type-of-change labels have been added.
- [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] Tests have been added and/or modified to cover the changes in this PR.
- [ ] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

File: data/cisagov_N.yml [hunk 1]
```diff
@@ -95,28 +95,28 @@ software:
     product: OptimalPlus
     cves:
       cve-2021-4104:
-        investigated: false
+        investigated: true
         affected_versions: []
```
Comment (Lcerkov) [inline data/cisagov_N.yml:99]:
```suggestion
        affected_versions:
          -Limited to deployments running Vertica, Cloudera, or Logstash
```

File: data/cisagov_N.yml [hunk 2]
```diff
@@ -95,28 +95,28 @@ software:
...
         fixed_versions: []
         unaffected_versions: []
       cve-2021-44228:
-        investigated: false
+        investigated: true
         affected_versions: []
```
Comment (Lcerkov) [inline data/cisagov_N.yml]:
```suggestion
        affected_versions:
          -Limited to deployments running Vertica, Cloudera, or Logstash
```

File: data/cisagov_N.yml [hunk 3]
```diff
@@ -95,28 +95,28 @@ software:
...
         fixed_versions: []
         unaffected_versions: []
       cve-2021-45046:
-        investigated: false
+        investigated: true
         affected_versions: []
```
Comment (Lcerkov) [inline data/cisagov_N.yml:110]:
```suggestion
        affected_versions:
          -Limited to deployments running Vertica, Cloudera, or Logstash
```

File: data/cisagov_N.yml [hunk 4]
```diff
@@ -95,28 +95,28 @@ software:
...
         fixed_versions: []
         unaffected_versions: []
       cve-2021-45105:
-        investigated: false
+        investigated: true
         affected_versions: []
```
Comment (Lcerkov) [inline data/cisagov_N.yml:115]:
```suggestion
        affected_versions:
          -Limited to deployments running Vertica, Cloudera, or Logstash
```

**Review by Lcerkov — CHANGES_REQUESTED** [review]
@kyle-ni The logic behind the calculated "Status" field requires that there be content in the affected version field for the Status to = Affected. I suggested adding the language in the notes section to the affected versions section.

**Comment by kyle-ni** [comment]
> 

@Lcerkov - Makes sense.  I missed how 'Status' was computed and the fact that it is an 'OR' of the individual CVEs.  Was attempting to communicate that we have investigated against all of the CVEs, but only vulnerable to one.

**Comment by Lcerkov** [comment]
@kyle-ni  Ahh, is it only vulnerable to CVE-2021-44228? If so, I will make that change and then merge the PR.

**Comment by kyle-ni** [comment]
> 

@Lcerkov Yes, that is correct.  Technically only CVE-2021-44228 applies for our products.

**Review by Lcerkov — APPROVED** [review]

**Comment by Lcerkov** [comment]
@kyle-ni Thanks for the clarification- I fixed that and merged the update!

**Pull request merged**